unic/char/range/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "unic-char-range"
+version = "0.5.0"
+authors = ["The UNIC Project Developers"]
+repository = "https://github.com/behnam/rust-unic/"
+license = "MIT/Apache-2.0"
+keywords = ["text", "unicode", "iteration"]
+description = "UNIC - CharRange"
```

Comment:
Would be better to use more informative language in description, as it's the only text that shows up on crates.io lists and item page, and allows better matching in search.

To follow the pattern from other UNIC components, it can be something like this:
```
description = "UNIC - Unicode Characters - Character Range and Iteration"
```